var/spack/repos/builtin/packages/py-astropy/package.py

```diff
@@ -51,6 +50,12 @@ class PyAstropy(PythonPackage):
     depends_on("py-extension-helpers", when="@5.1:", type="build")
     depends_on("pkgconfig", type="build")
 
+    depends_on("py-pytest@7:", type="test")
+    depends_on("py-pytest-doctestplus@0.12:", type="test")
+    depends_on("py-pytest-astropy-header@0.2.1:", type="test")
+    depends_on("py-pytest-astropy@0.10:", type="test")
+    depends_on("py-pytest-xdist", type="test")
+
     # Optional dependencies
```

Comment:
While you're at it can you update these too? I would also rename `+extras` to `+all`.

Reply:
Done.

While testing it, I came (again) across the problem that `py-arrow` does not build without the `parquet` variant enabled. I think this was not the case in the past. For this PR I just enabled it, but should we enable it by default (in a separate PR), or any other suggestions how to fix it?

Error was:
```
  CMake Error at $spack/opt/spack/linux-fedora37-x86_64/gcc-12.3.1/cmake-3.26.3-6tiskhudhvczqaclpe5jl7f6zsbwshol/share/cmake-3.26/Modules/CMakeFindDependencyMacro.cmake:76 (find_package):
    By not providing "FindParquet.cmake" in CMAKE_MODULE_PATH this project has
    asked CMake to find a package configuration file provided by "Parquet", but
    CMake did not find one.

    Could not find a package configuration file provided by "Parquet" with any
    of the following names:

      ParquetConfig.cmake
      parquet-config.cmake

    Add the installation prefix of "Parquet" to CMAKE_PREFIX_PATH or set
    "Parquet_DIR" to a directory containing one of the above files.  If
    "Parquet" provides a separate development package or SDK, be sure it has
    been installed.
  Call Stack (most recent call first):
    $spack/opt/spack/linux-fedora37-x86_64/gcc-12.3.1/arrow-10.0.1-ruhwebjkp3pzmsned73octvoowvl4ukh/lib64/cmake/ArrowDataset/ArrowDatasetConfig.cmake:55 (find_dependency)
    CMakeLists.txt:121 (find_package)
```

Reply:
`py-arrow`, `py-pyarrow`, or `arrow`? It looks like arrow to me.

It should definitely be in the package that fails to build, not this package.

Reply:
Sorry, I mistyped in my comment above. It is `py-pyarrow` that throws the error when building because it seems to need arrow with the parquet variant enabled. `py-arrow` and `arrow` itself install fine on their own.

I have removed the parquet variant again, and this has to be fixed in a separate PR.